docs/configuration/selecting-excluding.rst

```diff
@@ -17,6 +17,9 @@ The ``select`` and ``exclude`` parameters are lists, with values like the follow
 - ``config.materialized:table``: include/exclude models with the config ``materialized: table``
 - ``path:analytics/tables``: include/exclude models in the ``analytics/tables`` directory
 - ``+node_name+1`` (graph operators): include/exclude the node with name ``node_name``, all its parents, and its first generation of children (`dbt graph selector docs <https://docs.getdbt.com/reference/node-selection/graph-operators>`_)
+- ``+/path/to/model_g+`` (graph operators): include/exclude all the nodes in the path ``/path/to/model_g``, their parents and children
```

Comment:
just trying to understand here if the path needs a leading slash and what would it mean. a leading slash would usually mean it's an absolute path, no? does it have to be a relative path `+path/to/model_g+` instead of the current `+/path/to/model_g+` to not include the leading slash?

Reply:
@pankajkoti This example uses an absolute path, but it should also work with relative paths if users do not add the leading slash:
https://github.com/astronomer/astronomer-cosmos/blob/967dd17bc91fe297a58c54f87c07fad5bb921d05/cosmos/dbt/selector.py#L95-L97

Reply:
I made some improvements to the descriptions so this is clearer: 9a5eaf8b758dd4872f86eff695cb290b36b00920, 0f08af2602b08ac1bf18fc164c5b5729de4bca0a